tests/Functional/PluginManagerCommandsTest.php

```diff
@@ -17,26 +17,42 @@ class PluginManagerCommandsTest extends TestCase
 
     protected const HELLO_COMMAND = 'hello';
     protected const PLUGIN_NAME = 'terminus-plugin-example';
+    protected const PLUGIN_NAMES = [
+        'terminus-plugin-example',
+        'terminus-composer-plugin',
+        'terminus-rsync-plugin',
+    ];
     protected const PLUGIN_PACKAGE = 'pantheon-systems/terminus-plugin-example';
+    protected const PLUGIN_PACKAGES = [
```

Comment:
Consider combining with PLUGIN_NAMES; either assume that the org is always "pantheon-systems", or split the name out of the package. Optional.

Reply:
Done.